metadata.json

```diff
@@ -59,6 +63,13 @@
         "8",
         "9"
       ]
+    },
+    {
+      "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+        "20.04",
+        "22.04"
```

Comment:
bolt is still missing for Debian 12 / Ubuntu 24.04 :(
* https://github.com/puppetlabs/bolt/issues/3280
* https://github.com/puppetlabs/bolt/issues/3281